node/pkg/watchers/solana/client.go

```diff
@@ -881,7 +876,7 @@ func (s *SolanaWatcher) fetchMessageAccount(ctx context.Context, logger *zap.Log
 			zap.Uint64("slot", slot),
 			zap.String("commitment", string(s.commitment)),
 			zap.Stringer("account", acc))
-		return true
+		return 0, true
```

Comment:
Since you're using named return values, you could just use the word `return` here and in all the other cases in this function, if you like. Either way is fine, of course.